core/src/chain/entry.rs

```diff
@@ -4,11 +4,18 @@ use std::hash::{Hash as _Hash, Hasher};
 #[derive(Clone, Debug, PartialEq, Serialize, Deserialize)]
 pub struct Entry {
     content: String,
+
+    // @TODO do NOT serialize entry_type in Entry as it should only be in Header
+    // @see https://github.com/holochain/holochain-rust/issues/80
+    entry_type: String,
 }
 
 impl _Hash for Entry {
```

Comment:
Since the standard `Hash` trait doesn't clash with anything in this context, could you remove the alias for now?

Reply:
it clashes with `Entry::hash()` doesn't it?

Reply:
It shouldn't. Not in this particular context.

Reply:
@sphinxc0re it causes recursion issues if i try `self.has()` but i can drop the alias, check out the latest commit